packing.gradle

```diff
@@ -275,7 +275,7 @@ task dockerImage(type: Exec) {
  * Build a docker container image with the current snapshot
  * DO NOT CALL IT DIRECTLY! Use the `make dockerPack` command instead
  */
-task dockerPack(type: Exec, dependsOn: ['pack']) {
+task dockerPack(type: Exec, dependsOn: ['packOne']) {
```

Comment:
Not sure about this, what's the need?

Reply:
This change isn't strictly necessary, just a minor optimization since it doesn't depend on the packAll task.

The other changes are necessary to make the dockerPack work.

Reply:
Fair enough